packages/atomic/src/components.d.ts

```diff
@@ -205,11 +205,13 @@ export namespace Components {
     }
     interface AtomicFoldedResultList {
         /**
-          * TODO:
+          * The name of the field that uniquely identifies a result within a collection.
+          * @defaultValue `foldingchild`:
```

Comment:
```suggestion
          * @defaultValue `foldingchild`
```

Reply:
fyi components.d.ts will automatically generate from the rest of the source code

Reply:
I'll do this one locally and push!